spring-cloud-function-kotlin/src/main/java/org/springframework/cloud/function/context/config/KotlinLambdaToFunctionAutoConfiguration.java

```diff
@@ -16,21 +16,9 @@
 
 package org.springframework.cloud.function.context.config;
 
-import java.lang.reflect.ParameterizedType;
-import java.lang.reflect.Type;
-import java.util.function.Consumer;
-import java.util.function.Function;
-import java.util.function.Supplier;
-
-
-import kotlin.jvm.functions.Function0;
-import kotlin.jvm.functions.Function1;
-import kotlin.jvm.functions.Function2;
-import kotlin.jvm.functions.Function3;
-import kotlin.jvm.functions.Function4;
+import kotlin.jvm.functions.*;
```

Comment:
We generally do not allow * imports, so the build actually fails with checkstyle violations. 
Just run `./mvnw clean install` and you'll see the violations. Our formatting details and plugins for intellij and eclipse are [here](https://github.com/spring-cloud/spring-cloud-build#code-conventions-and-housekeeping)

Reply:
Done